RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -136,6 +137,53 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     }
   }  // namespace t3dnn
 
+  namespace pt3dnn {
+
+    template <typename TAcc>
+    ALPAKA_FN_ACC ALPAKA_FN_INLINE bool runInference(TAcc const& acc,
+                                                     const float pt3_rPhiChiSquared,
```

Comment:
```suggestion
                                                     const float rPhiChiSquared,
```
is `pt3_` prefix really necessary?